selfdrive/car/tests/test_car_interfaces.py

```diff
@@ -45,6 +46,7 @@ def get_fuzzy_car_interface_args(draw: DrawType) -> dict:
   return params
 
 
+@pytest.mark.forked
```

Comment:
```suggestion
```

this isn't needed after https://github.com/commaai/opendbc/pull/1007

Reply:
We should also be conservative using it as it would have masked this

also found this, relevant? https://github.com/pytest-dev/pytest-forked/issues/87

Reply:
>We should also be conservative using it
I feel like we want it everywhere so we don't get segfaults in the unittests without any idea of which test they came from